app/services/airrecord_table.rb

```diff
@@ -27,7 +27,7 @@ def faraday_connection
           ActiveSupport::Cache.lookup_store(
             *Rails.configuration.cache_store,
             namespace: 'airtable',
-            expires_in: 300 # 5 minutes in seconds
+            expires_in: 3.hours
```

Comment:
Part of me wants to do this another way ... like with a Sidekiq job or something. Or extend the time period even farther out. Open to suggestions.

Reply:
This seems fine to me, one request to airtable per instance every three hours should be something they can handle, y'know.